### PR TITLE
Update Panel header style and minor updates

### PR DESCRIPTION
**Pull request by ezequielc**
## Description
Update Panel header style and minor updates.

- Template selector name ellipsis
- Fix cursor for drag handle
- Remove unneeded .playlist-item-container
- Changed Edit links to blue to match Back button

## Motivation and Context
Playlist UI

## How Has This Been Tested?
Locally
[stage-1]

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
If any Release Checklist items were intentionally skipped, please provide which ones and the reasons why

### CODE REVIEW

File: src/partials/template-editor/attribute-list.html [hunk 1]
```diff
@@ -9,10 +9,9 @@
       </span>
     </div>
     <div class="col-xs-2 pr-0 align-right">
-      <a id="branding-edit" ng-click="componentsFactory.editComponent(brandingComponent);">
-        <span class="attribute-edit text-nowrap">
```
Comment (alex-deaconu) [inline src/partials/template-editor/attribute-list.html:13]: @ezequielc can the `attribute-edit` class be deprecated in this case? Thanks.

**Review by alex-deaconu — APPROVED** [review]
👍

File: src/partials/template-editor/attribute-list.html (hunk 1 above)
Comment (ezequielc) [reply src/partials/template-editor/attribute-list.html:13]: @alex-deaconu Yes, I will remove it in my next PR. Thanks

Comment (alex-deaconu) [reply src/partials/template-editor/attribute-list.html:13]: @ezequielc sounds good 👍

**Pull request merged**